src/simulation.hpp

```diff
@@ -912,7 +913,7 @@ template <typename problem_t> void AMRSimulation<problem_t>::ellipticSolveAllLev
 		}
 
 		// solve Poisson equation with open b.c. using the method of James (1977)
-		amrex::Vector<amrex::MultiFab> phi(finest_level + 1);
+		phi.resize(finest_level + 1);
```

Comment:
I think the problem is that the first plotfile is output before the first timestep, so `phi` is not yet initialized the first time you access it. This code (plus the `phi[lev].define` and `phi[lev].setVal(0)` code below) needs to be called after the initial conditions are set, but before the first output is done.

Reply:
It should go here: https://github.com/quokka-astro/quokka/pull/406/files#diff-1643e903f8d7916aa711335f79a095b83a7aa844db8dd37ba137dbc62a5eafbeR579